Makefile

```
@@ -6,13 +6,16 @@ all: build dist
 build:
 	$(MAKE) -C src
 
-dist:
+dist: build
 	mkdir -p $(DIST)/build
 	cp -r $(SRC)/*.html $(SRC)/*.svg $(SRC)/term.js src/examples $(DIST)
```

SVG is bundled so no need to copy it to dist.

```suggestion
	cp -r $(SRC)/*.html $(SRC)/term.js src/examples $(DIST)
```